packages/wrangler/src/index.ts

```diff
@@ -204,33 +210,34 @@ export function createCLIParser(argv: string[]) {
 		})
 		.scriptName("wrangler")
 		.wrap(null)
+		.locale("en_US")
```

Comment:
forcing locale here, since we don't support anything other than `en`. This also ensures we don't run into any unexpected behaviour when we call `wrangler.updateStrings()` further down